### ISSUE TITLE
bash autocomplete

### ISSUE THREAD

**Issue by NickeZ** [tummychow/git-absorb#23]
Hi, thanks for this very nice tool! After installing it with `cargo install git-absorb` I don't get any bash completions and `git absorb --help` fails.

To remedy the `--help` situation I ran did the following:

```
help2man git-absorb > .local/share/man/man1/git_absorb.1
mandb
```

Now that I know that `-r` exists, the autocompletion is not so important anymore...

**Comment by tummychow** [comment]
right so there's two issues here:

- when you run `git absorb --help` (with a space between `git` and `absorb`), git will try to look up the manpage for `git-absorb`, which doesn't exist (#2). if you run `git-absorb --help` (without the space), then you'll actually see git-absorb's existing help output, but right now that doesn't go into a manpage where git would find it. so i have to add that
- no bash completion. i might be able to wrangle clap into generating this

**Comment by vimpostor** [comment]
Looks like this issue can be closed since #36 was merged.
![Screenshot_20221116_192442](https://user-images.githubusercontent.com/21310755/202262347-33a3787d-9595-4889-aa56-d66cf7aa7f5e.png)
